sklearn/decomposition/_kernel_pca.py

```diff
@@ -141,12 +179,22 @@ class KernelPCA(TransformerMixin, BaseEstimator):
         and Klaus-Robert Mueller. 1999. Kernel principal
         component analysis. In Advances in kernel methods,
         MIT Press, Cambridge, MA, USA 327-352.
+
+    For eigen_solver == 'arpack', refer to `scipy.sparse.linalg.eigsh`.
+
+    For eigen_solver == 'randomized', see:
+        `Finding structure with randomness: Stochastic algorithms
+        for constructing approximate matrix decompositions Halko, et al., 2009
+        (arXiv:909)`
+        `A randomized algorithm for the decomposition of matrices
+        Per-Gunnar Martinsson, Vladimir Rokhlin and Mark Tygert`
```

Comment:
Single back-ticks are reserved to quote variable names or other programming symbols. Let's just do:

```suggestion
        Finding structure with randomness: Stochastic algorithms
        for constructing approximate matrix decompositions Halko, et al., 2009
        (arXiv:909)
        A randomized algorithm for the decomposition of matrices
        Per-Gunnar Martinsson, Vladimir Rokhlin and Mark Tygert
```

Single back-ticks are also used to format reST links and references be we don't want to have complex link formatting in docstrings.